src/csvcubedmodels/rdf/dcat.py

```diff
@@ -119,6 +119,60 @@ def __init__(self, uri: str):
         Resource.__init__(self, uri)
         self.rdf_types.add(DCAT.Dataset)
 
+#The new class would be created here which also inherits the Resource class
+class Distribution(Resource):
+
+    isDistributionOf: Ann[
+        str, Triple(DCAT.isDistributionOf, PropertyStatus.recommended, URIRef)
+    ]
+    #these two will need changing probably
+    access_service: Ann[str, Triple(DCAT.accessService, PropertyStatus.mandatory, URIRef)]
+    access_url: Ann[str, Triple(DCAT.accessURL, PropertyStatus.mandatory, URIRef)]
+
+    byte_size: Ann[str, Triple(DCAT.byteSize, PropertyStatus.mandatory, Literal)]
+    compress_format: Ann[str, Triple(DCAT.compressFormat, PropertyStatus.recommended, URIRef)]
+    download_url: Ann[str, Triple(DCAT.downloadURL, PropertyStatus.recommended, URIRef)]
+    media_type: Ann[str, Triple(DCAT.mediaType, PropertyStatus.recommended, URIRef)]
+
+    #not sure about this
+    package_format: Ann[str, Triple(DCAT.packageFormat, PropertyStatus.recommended, URIRef)]
+
+    spatial: Ann[str, Triple(DCTERMS.spatial, PropertyStatus.recommended, URIRef)]
+    spatial_resolution_in_meters: Ann[
+        float,
+        Triple(
+            DCAT.spatialResolutionInMeters,
+            PropertyStatus.optional,
+            lambda l: Literal(l, XSD.decimal),
+        ),
+    ]
+
+    temporal: Ann[str, Triple(DCTERMS.temporal, PropertyStatus.recommended, URIRef)]
+    temporal_resolution: Ann[
+        str,
+        Triple(
+            DCAT.temporalResolution,
+            PropertyStatus.optional,
+            lambda l: Literal(l, XSD.duration),
+        ),
+    ]
+
+    conforms_to: Ann[str, Triple(DCTERMS.conformsTo, PropertyStatus.recommended, URIRef)]
+    description: Ann[str, Triple(DCTERMS.description, PropertyStatus.recommended, Literal)]
```

Comment:
Shouldn't this use `map_str_to_en_literal` again?

Reply:
I thought it is using a literal (as I took it form the documentation)

Reply:
but yeah I see why it's not correct this way